controllers/course.js

```diff
@@ -0,0 +1,102 @@
+const course = {
+    name: "tema 1",
+    endpoint: "/tema1",
+    content: [
+        {
+            id: 1,
+            title: "El nombre es el propósito/Hasta Jacob.",
+            description:
+                "1 dentidad ¿quiénes somos? ¿De quiénes venimos? ¡Conoce la historia de esta familia de la cuál tú eres parte!",
```

Comment:
"1 dentidad" review this...

Reply:
@DanielCalleCO this data is temporary and in order to make the front end dynamic